.github/workflows/nightly.yml

```diff
@@ -120,7 +120,7 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.NODAI_INVOCATION_TOKEN }}
       with:
         release_id: ${{ steps.create_release.outputs.id }}
-        assets_path: ${GITHUB_WORKSPACE}/wheelhouse/nodai_*.whl
```

Comment:
why?

Reply:
when we updated assets path last time it hasnt published with .whl so it may not be finding the wheels correctly. This may fix the path by removing the brackets

Reply:
it hasn't fixed it but will in follow-up patch as this has more urgent fixes